compat/Makefile.am

```diff
@@ -29,14 +29,6 @@ libcompatsquid_la_SOURCES = \
 	eui64_aton.c \
 	eui64_aton.h \
 	fdsetsize.h \
-	getaddrinfo.cc \
-	getaddrinfo.h \
-	getnameinfo.cc \
-	getnameinfo.h \
-	inet_ntop.cc \
-	inet_ntop.h \
-	inet_pton.cc \
-	inet_pton.h \
 	initgroups.h \
```

Comment:
I have adjusted PR title/description. I started that edit to fix M-failed-description, but then discovered that the PR removes more than "functions" and edited some more. Please review and adjust further as needed.

> Adjusted PR description: All modern network stacks provide inet_ntop(), inet_pton(),
getaddrinfo(), and getnameinfo() APIs.

The code remaining in compat/os/opensolaris_10_netdb.h still defines `struct addrinfo`, `EAI_OVERFLOW`, and various other getaddrinfo() API components (see commit 3729717). PR description can be interpreted as implying that Squid only supports "modern network stacks". Based on that assumption, we should remove all those Solaris replacements, possibly undoing the entire hack introduced in commit 3729717 and, hence, effectively dropping Solaris 9 support (if it is still considered supported by anyone).

I do not insist on any changes.

Reply:
Solaris 9 is not supported by Oracle anymore; I'll remove

Reply:
Removed

Reply:
Glad those hacks can be removed now. I added the corresponding note to the PR description.